utils/prop-types/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@espressive/prop-types",
+  "version": "1.0.0",
+  "description": "Extended react custom props",
+  "homepage": "https://github.com/Espressive/cascara#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Espressive/cascara.git",
+    "directory": "utils/prop-types"
+  },
+  "license": "MIT",
+  "author": "Rocio Baena rbaena@espressive.com",
+  "main": "src/index.js",
+  "files": [
+    "index.js",
+    "src"
+  ],
+  "scripts": {
+    "prepare": "echo 'No prepare scripts for this package'"
+  }
+}
```

Comment:
This package should have a `peerDependency` defined as `prop-types` as it will need `prop-types` in order to work. The reason we probably cannot just include `prop-types` is that some libraries also require `prop-types` as a `peerDepencency` and those packages would not know that our custom prop types already have it.

We _also_ need to include `prop-types` as a `devDepencency` here. Any node module that we use in this package needs to be declared as a dependency in the package.json file. We use `prop-types` on line 1 of src/prop-types.js

Reply:
@brianespinosa @lordpixel can you look at this change, please? I understand of the peerDependency, but I'm not sure in I established the devDepency correctly:
`  "devDependencies": {
    "prop-types": "15.7.2"
  },
  "peerDependencies": {
    "prop-types": "15.7.2"
  }`

Reply:
@rociobaena LGTM 👍